projects/workflows-creator/src/lib/builder/builder.component.ts

```diff
@@ -60,8 +58,9 @@ export class BuilderComponent<E> implements OnInit, OnChanges {
     private readonly elements: ElementService<E>,
     private readonly cdr: ChangeDetectorRef,
     private readonly localizationSvc: LocalizationProviderService,
-  ) {}
-
+  ) {
+    this.localizationSvc.setLocalizedStrings(this.localizedStringMap);
```

Comment:
arre why? you are doing this in onInit na?

Reply:
sir its in Constructer, I had to remove this from ngonint n put in into the constructer bcos it was not working as expected , the Default strings were redendered on UI but not the localized one.